releasetasks/test/test_source.py

```diff
@@ -0,0 +1,146 @@
+            "queue:create-task:aws-provisioner-v1/build-c4-2xlarge",
+            "docker-worker:cache:build-foo-release-workspace",
+            "docker-worker:cache:tooltool-cache",
+            "signing:format:gpg",
+            "signing:cert:release-signing",
+            "docker-worker:relengapi-proxy:tooltool.download.public"
+        ])
+        assert expected_graph_scopes.issubset(self.graph["scopes"])
+
+    def test_task_scopes(self):
+        expected_task_scopes = set([
+            "docker-worker:cache:tc-vcs",
+            "docker-worker:image:taskcluster/builder:0.5.9",
+            "queue:define-task:aws-provisioner-v1/opt-linux64",
+            "queue:create-task:aws-provisioner-v1/opt-linux64",
+            "queue:define-task:aws-provisioner-v1/build-c4-2xlarge",
+            "queue:create-task:aws-provisioner-v1/build-c4-2xlarge",
+            "docker-worker:cache:build-foo-release-workspace",
+            "docker-worker:cache:tooltool-cache",
+            "docker-worker:relengapi-proxy:tooltool.download.public",
+        ])
+        assert expected_task_scopes.issubset(self.task["scopes"])
+
+    def test_signing_task_requirements(self):
+        assert self.signing_task_def["requires"][0] == self.task_def["taskId"]
+
+    def test_signing_task_provisioner(self):
+        assert self.signing_task["provisionerId"] == "signing-provisioner-v1"
+
+    def test_signing_task_worker_type(self):
+        assert self.signing_task["workerType"] == "signing-worker-v1"
+
+    def test_signing_task_scopes(self):
+        expected_task_scopes = set([
+            "signing:format:gpg",
+            "signing:cert:release-signing"
+        ])
+        assert expected_task_scopes.issubset(self.signing_task["scopes"])
+
+    def test_signing_manifest(self):
+        assert "signingManifest" in self.signing_task["payload"]
+
+    def test_signing_task_payload_length(self):
+        assert len(self.signing_task["payload"]) == 1
+
+    def test_required_graph_scopes(self):
+        graph = make_task_graph(
+            version="42.0b2",
+            appVersion="42.0",
+            buildNumber=3,
+            branch="foo",
+            revision="abcdef123456",
+            updates_enabled=False,
+            bouncer_enabled=False,
+            source_enabled=False,
+            en_US_config={"platforms": {
+                "linux": {"task_id": "xyz"},
+                "win32": {"task_id": "xyy"}
+            }},
+            l10n_config={},
+            verifyConfigs={},
+            signing_pvt_key=PVT_KEY_FILE,
+        )
+
+        do_common_assertions(graph)
+        self.assertEqual(graph["tasks"], None)
+
+        expected_scopes = set([
+            "signing:format:gpg",
+            "queue:define-task:buildbot-bridge/buildbot-bridge",
+            "queue:create-task:buildbot-bridge/buildbot-bridge",
+            "queue:task-priority:high",
+        ])
+        self.assertTrue(expected_scopes.issubset(graph["scopes"]))
```

Comment:
does this test really need its own task graph thats not common to the other, or was this just a mistaken leave-in?